core/tests/protractor_desktop/playTranslations.js

```diff
@@ -0,0 +1,103 @@
+ */
+
+var forms = require('../protractor_utils/forms.js');
+var general = require('../protractor_utils/general.js');
+var users = require('../protractor_utils/users.js');
+var workflow = require('../protractor_utils/workflow.js');
+
+var ExplorationEditorPage =
+    require('../protractor_utils/ExplorationEditorPage.js');
+var CreatorDashboardPage =
+    require('../protractor_utils/CreatorDashboardPage.js');
+var ExplorationPlayerPage =
+    require('../protractor_utils/ExplorationPlayerPage.js');
+var LibraryPage =
+    require('../protractor_utils/LibraryPage.js');
+
+describe('Test Translations', function() {
+  var creatorDashboardPage = null;
+  var explorationEditorPage = null;
+  var explorationEditorMainTab = null;
+  var explorationEditorTranslationTab = null;
+  var explorationEditorSettingsTab = null;
+  var explorationPlayerPage = null;
+  var libraryPage = null;
+
+  beforeAll(async function() {
+    creatorDashboardPage = new CreatorDashboardPage.CreatorDashboardPage();
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationEditorMainTab = explorationEditorPage.getMainTab();
+    explorationEditorTranslationTab =
+      explorationEditorPage.getTranslationTab();
+    explorationEditorSettingsTab = explorationEditorPage.getSettingsTab();
+    explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+    libraryPage = new LibraryPage.LibraryPage();
+
+    await users.createUser('testTranslations@translations.com',
+      'testTranslations');
+    await users.login('testTranslations@translations.com');
+    await workflow.createExploration();
+    await explorationEditorMainTab.exitTutorial();
+    await explorationEditorMainTab.setStateName('First');
+    await explorationEditorMainTab.setContent(await forms.toRichText(
+      'This is the first card.'));
+    await explorationEditorMainTab.setInteraction('EndExploration');
+    await explorationEditorPage.navigateToTranslationTab();
+    await explorationEditorTranslationTab.uploadAudioFileForLanguage(
+      'Hindi', '../data/cafe.mp3');
+    await explorationEditorPage.saveChanges();
+    await explorationEditorPage.navigateToSettingsTab();
+    await explorationEditorSettingsTab.setTitle('Test Translations');
+    await explorationEditorSettingsTab.setCategory('Languages');
+    await explorationEditorSettingsTab.setLanguage('English');
+    await explorationEditorSettingsTab.setObjective(
+      'Testing if translations works');
+    await explorationEditorPage.saveChanges('Done.');
+    await workflow.publishExploration();
+    await users.logout();
+  });
+
+  it('should play and pause audio translations', async function() {
+    await users.login('testTranslations@translations.com');
+    await libraryPage.get();
+    await libraryPage.playExploration('Test Translations');
+    await explorationPlayerPage.clickAudioBar();
+    await explorationPlayerPage.playAudio();
```

Comment:
Will this play the Hindi voiceover? Since the exploration language is "English", I would expect the default audio to be the auto-generated English voiceover, right?

Reply:
I tested it out locally, and the Hindi voiceover played by default.

Reply:
okay, sounds good!